lib/ansible/modules/cloud/docker/docker_config.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+#
+# Copyright 2016 Red Hat | Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: docker_config
+
+short_description: Manage docker configs.
+
+version_added: "2.6"
+
+description:
+     - Create and remove Docker configs in a Swarm environment. Similar to `docker config create` and `docker config rm`.
+     - Adds to the metadata of new configs 'ansible_key', an encrypted hash representation of the data, which is then used
+     - in future runs to test if a config has changed.
```

Comment:
I think the `-` in the front should not be there, since this is not a new bullet point, but the second part of the sentence started in the previous item.